docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/setup.md

```diff
@@ -0,0 +1,136 @@
+   ```bash
+   sudo groupadd docker
+   ```
+
+      <div class="admonition note">
+         <p class="admonition-title">Note</p>
+         <p>If the group already exists, `groupadd` outputs a message: **groupadd: group `docker` already exists**. This is OK.</p>
+      </div>
+      
+1. Add User to group:
+   ```bash
+   sudo usermod -aG docker $USER
+   ```
+
+1. Restart your computer for the changes to take effect.
+
+1. To verify the Docker installation, run <code>hello-world</code>:
+      ```bash
+      docker run hello-world
+      ```
+      A <strong>Hello from Docker!</strong> greeting indicates successful installation.
+
+      ```bash
+      Unable to find image 'hello-world:latest' locally
+      latest: Pulling from library/hello-world
+      2db29710123e: Pull complete 
+      Digest: sha256:10d7d58d5ebd2a652f4d93fdd86da8f265f5318c6a73cc5b6a9798ff6d2b2e67
+      Status: Downloaded newer image for hello-world:latest
+
+      Hello from Docker!
+      This message shows that your installation appears to be working correctly.
+      ...
+      ```
+
+
+### Install Docker Compose
+Install Docker Compose from the official repository as documented on the [Docker Compose](https://docs.docker.com/compose/install/linux/#install-using-the-repository) site.
+
+### Install Tools
+Install the build, media streaming, and parsing tools:
+
+```bash
+sudo apt install build-essential ffmpeg jq curl
+```
+
+### Tool Descriptions
+The table below lists command line tools this guide uses to help with EdgeX configuration and device setup.
+
+| Tool        | Description | Note |
+| ----------- | ----------- |----------- |
+| **curl**     | Allows the user to connect to services such as EdgeX. |Use curl to get transfer information either to or from this service. In the tutorial, use `curl` to communicate with the EdgeX API. The call will return a JSON object.|
+| **jq**   |Parses the JSON object returned from the `curl` requests. |The `jq` command includes parameters that are used to parse and format data. In this tutorial, the `jq` command has been configured to return and format appropriate data for each `curl` command that is piped into it. |
+| **base64**   | Converts data into the Base64 format.| |
+
+>Table 1: Command Line Tools
+
+## Download EdgeX Compose
+1. Clone the EdgeX compose repository:
+   ```bash
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```
+
+1. Navigate to the `edgex-compose` directory:
+   ```bash
+   cd edgex-compose
+   ```
+
+1. Checkout the Levski release:
+   ```bash
+   git checkout levski
+   ```
+
+      <div class="admonition note">
+         <p class="admonition-title">Note</p>
+         <p>The `levski` branch is the latest stable branch at the time of this update.</p>
+      </div>
+
+1. Navigate back to your home directory:
+      ```bash
+      cd ~
+      ```
+
+
+## Next Steps
+
+   [Default Images>](./deployment.md){: .md-button }
+   [Build Images>](./custom-build.md){: .md-button } 
```

Comment:
I recommend we limit this to just the Default images option. when testing this out, I accidentally clicked the Build Images one without even realizing there were two options (since this one was on the right, meaning "next" for my brain).

Also, I think that the best first step for users is to use the default images, and then if they want to look into the custom images they can proceed after.